lib/web_ui/dev/chrome_installer.dart

```diff
@@ -33,6 +35,17 @@ Future<BrowserInstallation> getOrInstallChrome(
 }) async {
   infoLog ??= io.stdout;
 
+  // When running on LUCI, if we specify the "chrome_and_driver" dependency,
+  // then the bot will download Chrome from CIPD and place it in a cache and
+  // set the environment variable CHROME_EXECUTABLE.
+  if (io.Platform.environment.containsKey(_chromeExecutableVar)) {
+    infoLog.writeln('Using Chrome from cache: ${io.Platform.environment[_chromeExecutableVar]}');
```

Comment:
nit: I'd replace the word "cache" with `$_chromeExecutableVar variable` as we could use it for other things, such as testing WasmGC on Chrome Canary and such.

Reply:
Done.